libs/pilight/protocols/generic/generic_label.c

```diff
@@ -1,19 +1,10 @@
 /*
 	Copyright (C) 2015 - 2019 CurlyMo & Niek
 
-	This file is part of pilight.
-
-	pilight is free software: you can redistribute it and/or modify it under the
-	terms of the GNU General Public License as published by the Free Software
-	Foundation, either version 3 of the License, or (at your option) any later
-	version.
-
-	pilight is distributed in the hope that it will be useful, but WITHOUT ANY
-	WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
-	A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
-
-	You should have received a copy of the GNU General Public License
-	along with pilight. If not, see	<http://www.gnu.org/licenses/>
+-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Why has the comment prefix changes?

Reply:
Because I thought you changed licensing from ``GNU General Public License`` to ``Mozilla Public Licence``

Reply:
I know, i was commenting on the `--` instead of the `/*`